Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,474 @@
+        {
+            var min = decimal.Zero;
+
+            try
+            {
+                min = inputArray[0];
+            }
+            catch (IndexOutOfRangeException)
+            {
+            }
+            var index = 0;
+            for (int i = 0; i < inputArray.Length; i++)
+            {
+                if (inputArray[i] < min)
+                {
+                    min = inputArray[i];
+                    index = i;
+                }
+            }
+            return index;
+        }
+
+        /// <summary>
+        /// Returns an formated output of currency (ex. -¤1, ¤4, ¤1002, -¤1001).
+        /// </summary>
+        private static string FormatCurrency(decimal currency)
+        {
+            const char negativeSymbol = '-';
+
+            var formatedCurrency = new StringBuilder();
+
+            if (currency < 0)
+            {
+                currency *= -1;
+                if (DefaultCurrencyLocation == 0)
+                    formatedCurrency.Append(negativeSymbol).Append(DefaultCurrencySymbol).Append(currency);
+                if (DefaultCurrencyLocation == 1)
+                    formatedCurrency.Append(DefaultCurrencySymbol).Append(" ").Append(negativeSymbol).Append(currency);
+                return formatedCurrency.ToString();
+            }
+
+            if (DefaultCurrencyLocation == 0)
+            {
+                formatedCurrency.Append(DefaultCurrencySymbol).Append(currency);
+            }
+
+            if (DefaultCurrencyLocation == 1)
+            {
+                formatedCurrency.Append(DefaultCurrencySymbol).Append(" ").Append(currency);
+            }
+
+            return formatedCurrency.ToString();
+        }
+
+        /// <summary>
+        /// Return formated output for multiple names in array (Kai, EarLington and Mihail).
+        /// </summary>
+        private static string FormatStringAndCommas(string[] validPeople)
+        {
+            var sb = new StringBuilder();
+
+            for (int i = 0; i < validPeople.Length; i++)
+            {
+                sb.Append(validPeople[i]);
+                if (i + 2 < validPeople.Length)
+                {
+                    sb.Append(", ");
+                }
+                else if (i + 1 < validPeople.Length)
+                {
+                    sb.Append(" and ");
+                }
+            }
+
+            return sb.ToString();
+        }
+
+        /// <summary>
+        /// Finds highest balance for a single person.
+        /// </summary>
+        private static decimal HighestBalanceForSinglePerson(string personAndBalance)
+        {
+            var balanceList = ConvertStringToDecimalArray(personAndBalance);
+            var highestBalanceIndex = FindDecimalArrayMax(balanceList);
+
+            try
+            {
+                var highestBalance = balanceList[highestBalanceIndex];
+                return highestBalance;
+            }
+            catch (Exception)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971928)